clex/driver.c

```diff
@@ -400,7 +400,13 @@ void reverse_toks (int idx)
   int matched = 0;
   int which = 0;
   int i;
+
+#ifdef _MSC_VER
+  int *saved = calloc(N, sizeof(int));
```

Comment:
no-- a bit of efficiency here isn't going to make any difference, let's just try to keep the code clean